yarn.lock

```diff
@@ -0,0 +1,3226 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file is irrelevant, please remove it from pull request.

Reply:
It's recommended to have `yarn.lock` in source control, from the [docs](https://yarnpkg.com/en/docs/migrating-from-npm),
>Yarn will generate a yarn.lock file within the root directory of your package. You don’t need to read or understand this file - just check it into source control. When other people start using Yarn instead of npm, the yarn.lock file will ensure that they get precisely the same dependencies as you have.

I can remove it if you think we don't need to have it in source control.

Reply:
1. We have always been using npm instead of yarn. Your can see there is a `.npmrc` in the repo.
2. This package has no `dependencies`, so really I'm not worried about version locking.
3. Either `package-lock.json` or `yarn.lock` does not affect users of the package.

https://docs.npmjs.com/files/package-lock.json
> One key detail about package-lock.json is that it cannot be published, and it will be ignored if found in any place other than the toplevel package.

https://yarnpkg.com/en/docs/yarn-lock
> Don’t worry about publishing the yarn.lock file as it won’t have any effect on users of the library.

So I'd prefer not having a ~3k lines file here :)